### PR TITLE
Turn on the contract actions sidecar in mainnet and testnet

### PR DESCRIPTION
**Pull request by david-bakin-sl**
…l envs

Signed-off-by: David Bakin <117694041+david-bakin-sl@users.noreply.github.com>**Description**:

**Related issue(s)**:

Fxes #5686 partially (the 0.37 part of it)

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-node/hedera-app/src/test/resources/bootstrap.properties [hunk 1]
```diff
@@ -185,7 +185,7 @@ hedera.recordStream.queueCapacity=5000
 hedera.recordStream.recordFileVersion=6
 hedera.recordStream.signatureFileVersion=6
 hedera.recordStream.logEveryTransaction=false
-hedera.recordStream.compressFilesOnCreation=false
+hedera.recordStream.compressFilesOnCreation=true
```
Comment (mhess-swl) [inline hedera-node/hedera-app/src/test/resources/bootstrap.properties:188]: Is `balances.compressOnCreation` intentionally set to `false` in this file? Or should it be updated to `true`?

File: hedera-node/test-clients/src/main/resource/bootstrap.properties [hunk 2]
```diff
@@ -186,7 +186,7 @@ hedera.recordStream.queueCapacity=5000
 hedera.recordStream.recordFileVersion=6
 hedera.recordStream.signatureFileVersion=6
 hedera.recordStream.logEveryTransaction=false
-hedera.recordStream.compressFilesOnCreation=false
+hedera.recordStream.compressFilesOnCreation=true
```
Comment (mhess-swl) [inline hedera-node/test-clients/src/main/resource/bootstrap.properties:189]: Same here, should `balances.compressOnCreation` be `true`?

**Review by mhess-swl — APPROVED** [review]

**Review by Nana-EC — DISMISSED** [review]

File: hedera-node/hedera-app/src/test/resources/bootstrap.properties (hunk 1 above)
Comment (david-bakin-sl) [reply hedera-node/hedera-app/src/test/resources/bootstrap.properties]: this is where I think I was expecting _your_ PR to land to change that.

File: hedera-node/test-clients/src/main/resource/bootstrap.properties (hunk 2 above)
Comment (david-bakin-sl) [reply hedera-node/test-clients/src/main/resource/bootstrap.properties]: and same here

File: hedera-node/hedera-app/src/test/resources/bootstrap.properties (hunk 1 above)
Comment (mhess-swl) [reply hedera-node/hedera-app/src/test/resources/bootstrap.properties]: Correct. I reviewed this before mine was merged

**Review by Nana-EC — APPROVED** [review]

**Review by qnswirlds — APPROVED** [review]

**Review by kimbor — APPROVED** [review]
lgtm!

**Pull request merged**